velox/expression/tests/ExpressionVerifier.cpp

```diff
@@ -17,16 +17,34 @@
 #include "velox/expression/tests/ExpressionVerifier.h"
 #include "velox/expression/Expr.h"
 #include "velox/vector/VectorSaver.h"
+#include "velox/vector/fuzzer/VectorFuzzer.h"
 
 namespace facebook::velox::test {
 
 namespace {
 
-void printRowVector(const RowVectorPtr& rowVector) {
-  LOG(INFO) << "RowVector contents (" << rowVector->type()->toString() << "):";
+// File names used to persist data required for reproducing a failed test case.
+static const char* kInputVectorFileName = "inputVector";
+static const char* kIndicesOfLazyColumnsFileName = "indicesOfLazyColumns";
+static const char* kResultVectorFileName = "resultVector";
```

Comment:
Maybe change file names to input_vector, result_vector, indices_of_lazy_columns to conform with the convention